torch/_inductor/compile_fx.py

```diff
@@ -821,10 +821,11 @@ def _compile_fx_inner(
                 },
                 payload_fn=lambda: json.dumps(cache_info),
             )
-
         compiled_graph.post_compile(example_inputs, cudagraphs, constants)
 
     log.debug("FX codegen and compilation took %.3fs", time.time() - start)
+    # Clear Compiled Triton Kernels per inductor compile
```

Comment:
can you add a comment explaining why this is important? emphasize it  should be done